ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -387,9 +387,8 @@ tests:
   steps:
     cluster_profile: equinix-ocp-metal
     workflow: baremetalds-e2e-virtualmedia
-- as: e2e-openstack
-  optional: true
-  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|images|pkg(/.+)?|cmd/openshift-install|terraform/providers)/(agent|alibabacloud|aws|azure|azurestack|baremetal|gcp|ibmcloud|libvirt|nutanix|ovirt|powervs|vsphere)/)
+- as: e2e-openstack-ovn
+  run_if_changed: openstack | (^data/.*/rhcos\.json)
```

Comment:
Do the spaces not cause problems with this regex?

Reply:
fixed, thx!